packages/patternfly-4/react-core/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/Alert/alert.css';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { AlertIcon } from './AlertIcon';
+import { capitalize } from '../../helpers/util';
+import { Omit } from '../../helpers/typeUtils';
+
+export enum AlertVariant {
+  success = 'success',
+  danger = 'danger',
+  warning = 'warning',
+  info = 'info'
+}
+
+export interface AlertProps
+  extends Omit<React.HTMLProps<HTMLDivElement>, 'action' | 'title'> {
+  /** Adds Alert variant styles  */
+  variant: 'success' | 'danger' | 'warning' | 'info';
+  /** Title of the Alert  */
+  title: React.ReactNode;
+  /** Action button to put in the Alert. Should be <AlertActionLink> or <AlertActionCloseButton> */
+  action?: React.ReactNode;
+  /** Content rendered inside the Alert */
+  children?: React.ReactNode;
+  /** Additional classes added to the Alert  */
+  className?: string;
+  /** Adds accessible text to the Alert */
+  'aria-label'?: string,
+  /** Variant label text for screen readers */
+  variantLabel?: string;
+};
+
+export const Alert: React.FunctionComponent<AlertProps> = ({
+  variant,
+  variantLabel,
+  'aria-label': ariaLabel = `${capitalize(variant)} Alert`,
+  action = null,
+  title,
+  children = '',
+  className = '',
+  ...props
+}: AlertProps) => {
+  const readerTitle = (
+    <React.Fragment>
+      <span className={css(accessibleStyles.screenReader)}>
+        {variantLabel || `${capitalize(variant)} alert:`}
```

Comment:
Why are you are concatenating the string here and not with he default props similar to aria-label?

Reply:
It's not concatenation. It's the same as
```js
variantLabel ? variantLabel : `${capitalize(variant)} alert:`
```

Reply:
I think it can be updated on line 36 to be 
```suggestion
        variantLabel = `${capitalize(variant)} alert:`
```
and changed here to:

```suggestion
        {variantLabel}
```

this way the consumers would know what we are defaulting too if they don't provide any variantLabel